content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -0,0 +1,431 @@
+   de trabalho adicionais não sejam escalonadas para executar neste nó.
+
+1. Gera o token que nós adicionais podem utilizar para associarem-se a uma
+   camada de gerenciamento no futuro. Opcionalmente, o usuário pode fornecer um
+   token através da opção `--token`, conforme descrito na documentação do
+   comando [kubeadm token](/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-token/).
+
+1. Prepara todas as configurações necessárias para permitir que nós se associem
+   ao cluster utilizando os mecanismos de
+   [Tokens de Inicialização](/pt-br/docs/reference/access-authn-authz/bootstrap-tokens/)
+   e [Inicialização TLS](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/):
+
+   - Escreve um ConfigMap para disponibilizar toda a informação necessária para
+     associar-se a um cluster e para configurar regras de controle de acesso
+     baseada em funções (RBAC).
+
+   - Permite o acesso dos tokens de inicialização à API de assinaturas CSR.
+
+   - Configura a auto-aprovação de novas requisições CSR.
+
+   Para mais informações, consulte
+   [kubeadm join](/docs/reference/setup-tools/kubeadm/kubeadm-join/).
+
+1. Instala um servidor DNS (CoreDNS) e os componentes adicionais do kube-proxy
+   através do servidor da API. A partir da versão 1.11 do Kubernetes, CoreDNS é
+   o servidor DNS padrão. Mesmo que o servidor DNS seja instalado nessa etapa,
+   o seu Pod não será escalonado até que um CNI seja instalado.
+
+   {{< warning >}}
+   O uso do kube-dns com o kubeadm foi descontinuado na versão v1.18 e removido
+   na versão v1.21 do Kubernetes.
+   {{< /warning >}}
+
+### Utilizando fases de inicialização com o kubeadm {#init-phases}
+
+O kubeadm permite que você crie um nó da camada de gerenciamento em fases
+utilizando o comando `kubeadm init phase`.
+
+Para visualizar a lista ordenada de fases e subfases, você pode rodar o comando
+`kubeadm init --help`. A lista estará localizada no topo da ajuda e cada fase
+tem sua descrição listada juntamente com o comando. Perceba que ao rodar o
+comando `kubeadm init` todas as fases e subfases são executadas nesta ordem
+exata.
+
+Algumas fases possuem flags específicas. Caso você deseje ver uma lista de todas
+as opções disponíveis, utilize a flag `--help`. Por exemplo:
+
+```shell
+sudo kubeadm init phase control-plane controller-manager --help
+```
+
+Você também pode utilizar a flag `--help` para ver uma lista de subfases de uma
+fase superior:
+
+```shell
+sudo kubeadm init phase control-plane --help
+```
+
+`kubeadm init` também expõe uma flag chamada `--skip-phases` que pode ser
+utilizada para pular a execução de certas fases. Esta flag aceita uma lista de
+nomes de fases. Os nomes de fases aceitos estão descritos na lista ordenada
+acima.
+
+Um exemplo:
+
+```shell
+sudo kubeadm init phase control-plane all --config=configfile.yaml
+sudo kubeadm init phase etcd local --config=configfile.yaml
+# você pode agora modificar os manifestos da camada de gerenciamento e do etcd
```

Comment:
think about that "agora você pode modificar" instead "você pode agora modificar".

Reply:
Yeah that indeed sounds better, just changed it!